Bhagyashri/Correction/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+     <head>
+     <title>Project First Page</title>
+	 <link rel="stylesheet" href="./css/style.css"/>
+     </head>
+	  <body>
+       <section class="first-sect1">
+	     <header>
+		    <hgroup class="class-hg1">
+	          <img src="./img/youtube.jpeg" alt="wesite logo"/>
+            </hgroup>
+		    <hgroup class="class-hg2">
+		      <nav>
+		       <a href="">home </a> /
+	           <a href="">about </a> /
+	           <a href="">our services</a> /
+               <a href="">our work </a> /
+	           <a href="">contact </a>
+	         </nav>
+	       </hgroup>
+		 </header>
+       </section>
+	 
+       <section class="class-h3">
+	    <h3 class="class-h31"> 
+	     I am David Husseye, a freelance Designer based in United Kingdom.I design interactive experience
+	    </h3>
+	    <h3 class="class-h32">
+	     Web+Mobile+Branding+Motion
+	    </h3>
+        </section>
+        <section>
+          <div class="class-left">
+	         <h1>Featured Project</h1>
+	         <h2>Project Name</h2>
+	      <div>
+	          <p>Lorem Ipsum is simply dummy text of the printing and<br/> 
+		         typesetting industry.Lorem Ipsum has been the industri's <br/>
+		         standard dummy text ever since.
+	          </p>
+		       <ul type="none">
```

Comment:
remove type attribute here. and add css attribute to ul
list-type: none;

Reply:
Have added list-style:none;